cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,229 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 4.1.1 <http://docs.cask.co/cdap/4.1.1/index.html>`__
+=============================================================
+
+New Features
+------------
+
+- :cask-issue:`CDAP-8872` - Changed the Apache Tephra dependency to the 0.11.0-incubating
+  version.
+
+- :cask-issue:`CDAP-9250` - Added support for the Hive version shipped with HDP 2.6.
+
+- :cask-issue:`CDAP-7731` - Added fetch size and transaction flush interval configurations
+  to the Kafka Consumer Flowlet.
```

Comment:
What is the Kafka Consumer Flowlet? Part of the kafka-pack? I don't think this is a new feature. Let's move it to improvements?